src/components/BestSellersCardList.astro

```diff
@@ -1,41 +1,121 @@
+    {title:'', price:''},
+    
+    {title: 'Peelingi kawitacyjne z ampułką i maską:', price: '170zł'},
+    {title: 'Kwasy i maska:', price: '100zł'}];
+
+
+    const bestSellersList2 = [
+  {title: 'Zabiegi na ciało', price: ''},
+  {title:'', price:''},
+
+    {title: 'Liposukcja ultradźwiękowa (strefa):', price: '150zł'},
+    {title:'', price:''},
+    
+  {title: 'Brwi oraz rzęsy', price: ''},
+  {title:'', price:''},
+
+    {title: 'Regulacja brwi:', price: '20zł'},
+    {title: 'Henna pudrowa z regulacją:', price: '80zł'},
+    {title: 'Henna rzęs:', price: '40zł'},
+    {title: 'Laminacja brwi:', price: '120zł'},
+    {title: 'Laminacja rzęs:', price: '130zł'},
+  {title:'', price:''},
+
+  {title: 'Rzęsy', price: ''},
+  {title:'', price:''},
+
+      {title: '1:1', price: '110-130zł'},
+      {title: '2D', price: '120-140zł'},
+      {title: '3D', price: '130-150zł'},
+      {title: '4D', price: '140-160zł'},
+      {title: '5D', price: '150-170zł'},
+      {title: '6D', price: '160-180zł'} ];
+    
+      const bestSellersList3 = [
+  {title: 'Wosk', price: ''},
+  {title:'', price:''},
+
+    {title: 'Górna warga:', price: '20zł'},
+    {title: 'Twarz:', price: '50zł'},
+    {title: 'Pachy:', price: '60zł'},
+    {title: 'Ręce:', price: '60zł'},
+  {title:'', price:''},
+
+      {title: 'Manicure klasyczny:', price: '50zł'},
+      {title: 'Manicure hybrydowy:', price: '60zł'},
+      {title: 'Manicure żelowy z przedłużeniem:', price: '90zł'},
+      {title: 'Zabieg pielęgnacyjny na dłonie:', price: '60zł'},
+      {title: 'Zabieg pielęgnacyjny na stopy:', price: '60zł'},
+];
+
+const boldTitles = [
+  'Zabiegi na twarz:',
+  'Zabiegi na ciało',
+  'Brwi oraz rzęsy',
+  'Rzęsy',
+  'Wosk',
 ];
+
+const pinkText = 'Zabiegi dopasowywuję do indywidualnych potrzeb skóry, po jej diagnozie, można je łączyć, zmieniać. Skontaktuj się ze mną, aby uzyskać spersonalizowane porady w sprawie pielęgnacji.';
 ---
 
 <ul class="flex mt-[40px] flex-wrap justify-between">
-  {
-    bestSellersList.map(({ title, description, price, image }) => (
-      <li class="w-[32%] cursor-pointer rounded-b-lg  shadow-lg hover:shadow-2xl">
-        <Image class="rounded-t-lg h-[250px]" src={image} alt="image" />
-        <div class="border border-pinkLight  border-t-0 px-[20px]  pb-[20px]">
-          <h3 class="font-bold mt-[15px]">{title}</h3>
-          <p class="mt-[10px] font-light">{description}</p>
-          <p class="sectionTitle mt-[20px]">$ {price}</p>
-        </div>
-      </li>
+  <li class="rounded-lg shadow-lg bg-white w-full mobile:w-full tablet:w-[48%] desktop:w-[32%]">
+    <ul>
+    {bestSellersList1.map(({ title, price}) => (
+  
+  <div class="flex justify-between px-[20px] pt-[10px] pb-[10px]">
+          <h3 class={`text-base mb-[1px] text-left ${boldTitles.includes(title) ? 'font-bold' : ''} ${title === pinkText ? 'text-accent' : ''}`}>{title}</h3>
```

Comment:
tu zamiast zwracac pusty string mozesz spróbwac tego zapisu: 

```suggestion
          <h3 class={`text-base mb-[1px] text-left ${boldTitles.includes(title) && 'font-bold' } ${title === pinkText && 'text-accent'}`}>{title}</h3>
```

Reply:
O! tylko tyle? to piękne i przydatne! Nawet działa ;) Poprawione.